src/domain/model/character.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+type Character struct {
+	Id       int      `json:"id"`
```

Comment:
By convention (most lints even show a warning) the Id fields should be renamed as ID